aidants_connect_web/templates/public_website/habilitation.html

```diff
@@ -75,7 +71,44 @@ <h2>Liste des structures éligibles à l’utilisation d’Aidants Connect</h2>
             Exemples : les travailleurs sociaux.
           </li>
         </ul>
-        <p><strong>Les aidants familiaux, les bénévoles et services civiques ne sont donc pas éligibles.</strong></p>
+        <p><strong>Les aidants familiaux, les bénévoles et services civiques ne sont donc pas éligibles. Pour plus d'informations, 
+          nous vous invitons à consulter <a href="https://aidantsconnect.beta.gouv.fr/faq/mandat/">notre FAQ.</a></strong></p><br>
+        
+        <h2>Calendrier du déploiement</h2>
+        
+        <p><strong>Le déploiement de l'outil va s'effectuer par vagues territoriales progressives selon le calendrier suivant :</p><strong>
+          
+          <ul>
+          <li>1ère vague : Mars - avril 2021 :
+          </ul>
```

Comment:
Plutôt que des listes (ul) d'un seul élément, on pourrait utiliser des titres (ici des h3 typiquement).
```suggestion
    <h3> vague : Mars - avril 2021</h3>
```